_posts/2016-04-06-Testing-Schemes.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Using Xcode's Schemes to run a subset of your tests"
+date: 2016-04-06 12:09
+author: orta
+categories: [mobile, xcode]
+---
+
+[Eigen](https://github.com/artsy/eigen) has hit the point where testing is a chore, I [have ideas](https://github.com/orta/life/issues/71) on how to improve this, based on [Code Injection](http://artsy.github.io/blog/2016/03/05/iOS-Code-Injection/), but they aren't fully fleshed out yet and are time-intensive.
```

Comment:
I think explaining _why_ testing is a chore is important. We know its because the app has grown in terms of size, complexity, and # of developers, but the reader might not.